ml-agents/mlagents/trainers/components/bc/module.py

```diff
@@ -36,14 +31,16 @@ def __init__(
         :param samples_per_update: Maximum number of samples to train on during each BC update.
         """
         self.policy = policy
-        self.current_lr = policy_learning_rate * strength
-        self.model = BCModel(policy, self.current_lr, steps)
+        self.current_lr = policy_learning_rate * settings.strength
+        self.model = BCModel(policy, self.current_lr, settings.steps)
         _, self.demonstration_buffer = demo_to_buffer(
-            demo_path, policy.sequence_length, policy.brain
+            settings.demo_path, policy.sequence_length, policy.brain
         )
 
-        self.batch_size = batch_size if batch_size else default_batch_size
-        self.num_epoch = num_epoch if num_epoch else default_num_epoch
+        self.batch_size = (
+            settings.batch_size if settings.batch_size else default_batch_size
```

Comment:
why does this need to be a tuple now?

Reply:
It's not a tuple, it's the way `black` likes to format long lines 😛